Dockerfile

```diff
@@ -5,6 +5,9 @@ EXPOSE 8000
 ENV PIP_DISABLE_PIP_VERSION_CHECK 1
```

Comment:
```
FROM python:3.10-slim-bullseye
```
Any reason we're on `buster`?

Reply:
That's Michał's choice, probably just a muscle memory ;). Changed to bullseye.